masakari-controller/controller/masakari_util.py

```diff
@@ -569,89 +606,91 @@ def _fetch_project_id(self):
 
         return projects[0].id
 
+    @log_process_begin_and_end.output_log
```

Comment:
Why do these methods have the decorator?  When the log level is debug, the decorator shows method name and its arguments and then the message in try-section shows same message again.